app/src/main/java/com/jerboa/ui/components/home/BottomNavActivity.kt

```diff
@@ -238,7 +236,7 @@ fun BottomNavActivity(
                         )
                     }
                 },
-            ) { _ ->
+            ) { padding ->
```

Comment:
This change was need to propagate the padding of the navigation bar (ours) else content hides behind it. We can't use the previous approach that applies it to the navhost, that causes weird issues